### PR TITLE
Change gtk+ documentation URL to correct one

### PR DESCRIPTION
**Pull request by dmitryvk**
"/documentation.php" returns 404

### CODE REVIEW

**Comment by alatiera** [comment]
Looks good, the gtk.org website was redone and it wasn't updated since. Thanks!

File: src/lib.rs [hunk 1]
```diff
@@ -19,7 +19,7 @@
 //!
 //! - [General `GLib` family types and object system overview](../glib/index.html)
 //!
-//! - [GTK+ documentation](http://www.gtk.org/documentation.php)
+//! - [GTK+ documentation](https://www.gtk.org/docs/)
```
Comment (alatiera) [inline src/lib.rs:22]: Maybe should also drop the `+` from the name since it got renamed to just GTK. Best be done in a separate MR/issue as its probably many more places.

**Comment by GuillaumeGomez** [comment]
Fixed in #1057.

**Comment by dmitryvk** [comment]
#1057 changes different link (link to gtk-rs in Readme.md); this PR fixes link to GTK website in src/lib.rs.

**Comment by GuillaumeGomez** [comment]
Oh indeed, sorry!

**Comment by GuillaumeGomez** [comment]
Thanks!
